brainstormforce/ultimate-addons-for-gutenberg
# Fixed: Multiple AJAX requests generated if Pagination Markup is returned empty.

### Description
Trello Task - https://trello.com/c/GVMmLsu3/130-post-pagination-issue


### Types of changes
Bugfix (non-breaking change which fixes an issue) 

### How has this been tested?
 - Add a CPT to your site.
 - Add one post of that CPT.
 - Take Post Grid block on a page and select that post type and update the post.
 - You will see too many AJAX requests in console > Network and site will become slow or crash. 

### Checklist:
